### ISSUE TITLE
TO DO (7/31/2023)

### ISSUE THREAD

**Issue by KY246** [KY246/jpdBlock#1]
Things to do:

- [x] Add Firefox Support
- [x] Add Extra Reviews
- [x] Allow multiple URLs to be blocked on one list without requiring regex
- [ ] <s>Add point support for bunpro</s>
- [ ] <s>Add point support for WaniKani (Low Priority)</s>
